homeassistant/components/unifi/sensor.py

```diff
@@ -108,6 +108,27 @@ def async_wlan_client_value_fn(hub: UnifiHub, wlan: Wlan) -> int:
     )
 
 
+@callback
+def async_device_clients_value_fn(hub: UnifiHub, device: Device) -> int:
+    """Calculate the amount of clients connected to a device."""
+
+    return len(
+        [
+            client.mac
+            for client in hub.api.clients.values()
+            if (
+                (
+                    client.access_point_mac != ""
+                    and client.access_point_mac == device.mac
+                )
+                or (client.access_point_mac == "" and client.switch_mac == device.mac)
+            )
+            and dt_util.utcnow() - dt_util.utc_from_timestamp(client.last_seen or 0)
+            < hub.config.option_detection_time
```

Comment:
Im not sure if we should consider this data for this sensor

Reply:
So my initial thought is that I want to know how many devices is on the network now.
If I understand it correctly, when removing the number will only increase when devices join the network right ?

Reply:
It would presumably just take a bit longer before the number changes

Reply:
Just to be clear and I fully understand it 
 will the number always increase and never decrease then?

Reply:
Remove it and test it out :)

I would assume it would decrease (but maybe in a lower fashion) as well but if it doesnt we keep it in

Reply:
Removed ti

Reply:
So you tester it and it still works?

Reply:
Sorry for back and forth here, I saw that the same thing was done on the wlan client side, so lets keep it as it was considering the last seen as well.